test/helpers/types/types.go

```diff
@@ -31,7 +31,8 @@ type ContainerLog struct {
 	Hostname         string                 `json:"hostname"`
 	PipelineMetadata PipelineMetadata       `json:"pipeline_metadata"`
 	LogType          string                 `json:"log_type,omitempty"`
-	ViaqIndexName    string                 `json:"viaq_index_name"`
+	ViaqIndexName    string                 `json:"viaq_index_name,omitempty"`
+	WriteIndex       string                 `json:"write_index,omitempty"`
```

Comment:
Why do we need "write" index?

Reply:
this is the field we used for vector, as the name `viaq-index-name` didnt make much sense for vector.
Also vector ES sink does not delete the key which contains the index name, as fluentd ES plugin can do. So if we keep `viaq-index-name` , it will be added to the record also.

Reply:
@alanconway any comments?